src/Routes/Devices/DeviceTable.js

```diff
@@ -213,6 +213,7 @@ const DeviceTable = ({
   fetchDevices,
   isSystemsView = false,
   isAddSystemsView = false,
+  ariaLabel,
```

Comment:
Passing the aria-label from `GroupsDetail` to `DropdownToggle` for the dropdown in the page header makes sense, but instead of drilling down through four intermediate components (`GroupsDetail` -> `DeviceTable` -> `GeneralTable` -> `ToolbarHeader` -> `ToolbarKebab` -> `KebabToggle`) for the other dropdown is a bit much. Using something like the `useContext()` hook might be better here.